### ISSUE TITLE
Disable some ts specific rules for js files

### ISSUE THREAD

**Issue by jimmywarting** [denoland/deno_lint#1437]
i stumble upon a issue where it says:

> This member must have an 'override' modifier because it overrides a member in the base class 'File'
> .deno-ts(4114)

well. i'm using plain vanilla js - so i can't use an override modifier.
( the error comes from the vscode deno extension - not that i'm executing any lint tooling )

i tried slamming a `@override` jsdoc to it, but no avail. still complain

minimal repo:
```js
class X extends File {

  /** @override */
  async text() {
    return ''
  }

}
```

Rule like this should not apply to `.js` files if it can't understand certain jsdoc features
